packages/rocketchat-lib/server/methods/filterBadWords.js

```diff
@@ -0,0 +1,25 @@
+
+var Filter = Npm.require('bad-words');
+    
+    RocketChat.callbacks.add( 'beforeSaveMessage' , function(message){
+        // console.log(message)
+        
+        if( RocketChat.settings.get('Message_AllowBadWordsFilter') )
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 6 space characters but found 8.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2432869137/issues/source?bid=3199634&fileBranchId=3240346#l7)